framework/test_app/runners/k8s/k8s_base_runner.py

```diff
@@ -998,9 +998,12 @@ def _wait_pod_started(self, name, **kwargs) -> k8s.V1Pod:
         logger.info("Waiting for pod %s to start", name)
         self.k8s_namespace.wait_for_pod_started(name, **kwargs)
         pod = self.k8s_namespace.get_pod(name)
-        logger.info(
-            "Pod %s ready, IP: %s", pod.metadata.name, pod.status.pod_ip
-        )
+
+        if hasattr(pod.status, "pod_ip_s"):  # if running with dualstack support
+            pod_ip_s = pod.status.pod_ip_s
+        else:
+            pod_ip_s = pod.status.pod_ip
+        logger.info("Pod %s ready, IP: %s", pod.metadata.name, pod_ip_s)
```

Comment:
```suggestion
        if hasattr(pod.status, "pod_ip_s"):  # if running with dualstack support
            pod_ips = pod.status.pod_ip_s
        else:
            pod_ips = pod.status.pod_ip
        logger.info("Pod %s ready, IP: %s", pod.metadata.name, pod_ips)
```

It sucks that k8s calls it `pod_ip_s` - they generate APIs automatically. But no reason to keep their naming in  our local variables

Reply:
Removed the local variable and directly logging the fields in both if and else block